home-server-generator/src/main/resources/mustaches/server/pom.mustache

```diff
@@ -174,32 +87,13 @@
     </dependency>
 
   </dependencies>
-  <repositories>
-    <repository>
-      <id>sonatype-snapshots</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-    </repository>
-  </repositories>
+
   <properties>
-    <java.version>{{#java8}}1.8{{/java8}}{{^java8}}1.7{{/java8}}</java.version>
+    <java.version>11</java.version>
```

Comment:
Do we really need change it?

Reply:
I guess because now it works with java 11 so I specified it.

Reply:
As I mentioned above we can depend on libraries with java gte v1.5. And open API generator doesn't support java 11. So it would be better to leave it as is and generate only code for java 8.

Reply:
Got back 1.8 in java.version.